src/AppInstallerCommonCore/Settings.cpp

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "Public/winget/Settings.h"
+#include "Public/AppInstallerRuntime.h"
+#include "Public/AppInstallerStrings.h"
+
+namespace AppInstaller::Settings

```

Comment:
>AppInstaller [](start = 10, length = 12)

Just to confirm that we're still using AppInstaller as root namespace for new code and we'll rename once for all in the future?

Reply:
Yes, the namespace will litter the code with both if we do it piecemeal.  Placing new header files in a directory won't.